governance/votes/Executive vote - November 2, 2022.md

```diff
@@ -0,0 +1,96 @@
+- DAI streams for various now offboarded will be yanked.
+- The MKR stream for the now offboarded [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be yanked.
+- [Oasis.app](https://oasis.app) will be whitelisted on the rETH/USD oracle.
+- **258,000 DAI** will be transferred to the BlockTower Legal and Risk Work SPF multi-sig as a result of the successful [Special Purpose Fund (SPF) application](https://vote.makerdao.com/polling/QmcV2pMZ#vote-breakdown).
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Maker Teleport will be deployed with **1 Basis Point** fee, a **1000 DAI** Deposit Limit, and **1 million DAI** Debt Ceiling on [Starknet](https://starkware.co/starknet/).
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Maker Teleport Technical Deployment on Starknet
+As per the outcome of successful Governance Polls [here](https://vote.makerdao.com/polling/QmbWkTvW) and [here](https://vote.makerdao.com/polling/QmZxRgvG), [Maker Teleport (formerly known as Maker Wormhole)](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) will be deployed on [Starknet](https://starkware.co/starknet/), if this executive passes. The Maker Teleport parameters will be set as follows:
+* The Starknet Teleport Fee parameter will be set to **1 Basis Point**.
+* The Deposit Limit will be set to **1000 DAI**.
+* The [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to **1 million DAI**.
+
+Please review [MIP69](https://mips.makerdao.com/mips/details/MIP69) for details on Layer-2-to-Layer-1 Fast Withdrawals and the [discussion thread](https://forum.makerdao.com/t/request-for-poll-starknet-bridge-deposit-limit-and-starknet-teleport-fees/17187) on  Starknet parameters to inform your position on this issue before voting.
+
+### Offboarded Core Units - Yanking DAI and MKR Streams
+As per multiple successful offboarding proposals (linked below), several DAI and MKR streams will be yanked if this executive proposal passes. Note that when a token stream is yanked, it will prevent new tokens from accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.
```

Comment:
In this section we can be specific 😄 

```suggestion
As per multiple successful offboarding proposals (linked below), three DAI and one MKR streams will be cancelled (`yank`ed) if this executive proposal passes. 

When a token stream is `yank`ed, it will prevent new tokens from accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.
```